src/index.html

```diff
@@ -3,9 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>To Do List</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <section class="container">
+        <div class="header">
+            <h4 class="tody-title"> Today's To Do </h4>
+        </div>
+        <div class="todo-body">
+            <div class="mb-3">
+            <input type="text" class="add-list-input" id="add-list" placeholder="Add your list ... ">
+            </div>
+            <ul id="todo-list"> </ul>
+        </div>
+        <footer class="footer"> <button class="clear-all-tasks"> Clear All Completed </button> </footer>
+    </section>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/aff5a1d8c6ef0b8b6077eb9723114033691a75fc/todo-list/m1_list_structure.md?plain=1#L36), kindly let your design be a clone of that shown below:

Your design:
![image](https://github.com/akbarsazish/To-Do-List/assets/20554583/be519dc3-490b-4b8a-9625-49b07a218454)

Design guide:
![image](https://camo.githubusercontent.com/d04bee7a5b8f62cc6655dbec65c4d14583e4d78d0c434ddc070f6881bb9568fc/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067)